### PR TITLE
Api/movies

### PR DESCRIPTION
**Pull request by ghost**
Updated

### CODE REVIEW

File: src/BookYourShow.Api/Controllers/GenresController.cs [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BookYourShow.Api.Controllers
+{
+    [Route("api/[controller]")]
```
Comment (basilin) [inline src/BookYourShow.Api/Controllers/GenresController.cs]: follow api naming conventions, `api/[controller]` to `genere`

File: src/BookYourShow.Api/Controllers/LanguagesController.cs [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BookYourShow.Api.Controllers
+{
+    [Route("api/[controller]")]
```
Comment (basilin) [inline src/BookYourShow.Api/Controllers/LanguagesController.cs]: `api/[controller]` to `language`

File: src/BookYourShow.Api/Controllers/MoviesController.cs [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BookYourShow.Api.Controllers
+{
+    [Route("api/[controller]")]
```
Comment (basilin) [inline src/BookYourShow.Api/Controllers/MoviesController.cs]: `api/[controller]` to `movie`

File: src/BookYourShow.Api/Controllers/GenresController.cs (hunk 1 above)
Comment (ghost) [reply src/BookYourShow.Api/Controllers/GenresController.cs]: api names changed

File: src/BookYourShow.Api/Controllers/LanguagesController.cs (hunk 2 above)
Comment (ghost) [reply src/BookYourShow.Api/Controllers/LanguagesController.cs]: api names changed

File: src/BookYourShow.Api/Controllers/MoviesController.cs (hunk 3 above)
Comment (ghost) [reply src/BookYourShow.Api/Controllers/MoviesController.cs]: api names changed

**Pull request merged**